diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Git is a web based open source repository for developers to share code / projects.   It also has a feature to store version control after each commit.
 
```

Comment:
**Github** is a user interface (UI) for accessing git's version control capabilities for our project repositories.